game/games_test.go

```diff
@@ -0,0 +1,49 @@
+package game
+
+import (
+	_ "encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestFindGame(t *testing.T) {
+	games := NewGames()
+	games.AddGame("test")
+
+	game, err := games.findGame(1)
+	if err != nil {
+		t.Errorf("Failed to retrieve game: error returned")
+	}
+
+	if game.Name != "test" {
+		t.Errorf("Failed to retrieve correct game")
+	}
+}
+
+func TestAddGame(t *testing.T) {
+	games := NewGames()
+	games.AddGame("test")
+
+	if len(games.games) != 1 {
+		t.Errorf("Failed to add game")
+	}
+
+	if game := games.games[0]; game.Name != "test" {
+		t.Errorf("Failed to AddGame with correct data")
+	}
+}
+
+func TestShow(t *testing.T) {
+	games := NewGames()
+	games.AddGame("test")
+
+	// TODO: This is actually borked. The ID is persistent across tests., track ID for game and set here
```

Comment:
Moving the sequence to Games would help fix this. We could look at the last number in the sequence. This will probably also be fixed just by moving it out of global scope.